Pipfile

```diff
@@ -10,7 +10,7 @@ requests = "*"
 gunicorn = "*"
 attrs = "*"
 thoth-common = "*"
-thoth-storages = "*"
+thoth-storages = ">=0.0.33"
```

Comment:
This shouldn't be necessary - we want to keep things as recent as possible with kebechet so wildcard should be good enough.

Reply:
how do we enforce minimum versions?

Reply:
ACK